packages/react-core/src/components/BackgroundImage/__tests__/BackgroundImage.test.tsx

```diff
@@ -2,19 +2,27 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { BackgroundImage } from '../BackgroundImage';
 
-test('has default className and src URL applied to style', () => {
+test('renders with default className pf-v5-c-background-image', () => {
+  render(<BackgroundImage src="/image/url.png" data-testid="test-id" />);
+  expect(screen.getByTestId('test-id')).toHaveClass('pf-v5-c-background-image', { exact: true });
+});
+
+test('has src URL applied to style', () => {
   render(<BackgroundImage src="/image/url.png" data-testid="test-id" />);
 
-  const backgroundImage = screen.getByTestId('test-id');
-  const backgroundImageStyle = backgroundImage.getAttribute('style');
+  const backgroundImageStyle = screen.getByTestId('test-id').getAttribute('style');
```

Comment:
Nit: we could remove this variable altogether and just have the expectation be `expect(screen.getByTestId('test-id')).toHaveAttribute('style', '--pf-v5-c-background-image--BackgroundImage:/image/url.png')` (give or take a space between the CSS var and value)